### PR TITLE
enhanced_readme

### LINKED ISSUE #46: Enhance readme.md file  and add Learn.md file
Hey @dvjsharma  , A Readme file plays an important role in User/Contributor convenience . It is essential in any given repository because it serves as the primary source of information about the project. It provides an overview of the project's purpose, how to install and use it, and any necessary prerequisites. Additionally, the README often includes guidelines for contributing, links to documentation, and contact information for the maintainers.
Let's create a more descriptive , attention seeking README file including all guidelines .

Kindly assign this issue to me under GSSoC'24 .

### PR DESCRIPTION
**Pull request by aasthakourav20**
Fixes: #46 

Added enhanced readme.md file
Closes: #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,30 +109,34 @@ Ensure that `Node.js` and `MySQL` are installed on your machine.
     node server.js
     ```
 
-After completing these steps, you should have the Drawn2Shoe application up and running on your local machine.
+After completing these steps, you should have the Drawn2Shoe application up and running on your local machine. 🎉
+
+8). Make the required changes and add the changed file . 
+9). Open a pull request from your branch to the main repository.
+10). Provide a clear description of your changes and why they're valuable.
```
Comment (dvjsharma) [inline README.md]: These points are specific to contributors and should not be included in the `Installation` heading. Please remove.

**Review by dvjsharma — CHANGES_REQUESTED** [review]
Minor Changes, rest looks good.

File: README.md (hunk 1 above)
Comment (aasthakourav20) [reply README.md]: > These points are specific to contributors and should not be included in the `Installation` heading. Please remove.
Hey , you are asking just to put these points under another subheading ....or just remove these points ??

Comment (dvjsharma) [reply README.md]: Remove these points.

Comment (aasthakourav20) [reply README.md]: Hey , @dvjsharma , changes are done ! Kindly check.

**Comment by dvjsharma** [comment]
Changes looks good. Please use `git amend` command to squash the commits into one so we can merge the PR.

**Comment by aasthakourav20** [comment]
> Changes looks good. Please use `git amend` command to squash the commits into one so we can merge the PR.

Hey @dvjsharma , Kindly review the changes .

**Comment by dvjsharma** [comment]
I cannot see the squashed commits. Please merge the commits.

**Comment by aasthakourav20** [comment]
> I cannot see the squashed commits. Please merge the commits.



> I cannot see the squashed commits. Please merge the commits.

Screenshot (221)
I used these command to merge commits  but unable to find where I did mistakes , would you please help.

**Comment by dvjsharma** [comment]
No worries, I think I can do it from my end as well. As a general note, do study about `amend` commands. Many open source organizations have this as a convention.

**Pull request merged**